README.md

```diff
@@ -239,7 +239,16 @@ docker run --restart=always \
        snyk/broker:github-com
 ```
 
-### Misc
+### Mounting Secrets
+Sometime it is required to load sensitive configurations (GitHub/Snyk's token) from a file instead from environment variables. Broker is using [dotenv](https://www.npmjs.com/package/dotenv) to load the config, so the process is relatively simple:
+* Create a file named `.env` and put your sensitive config there:
+```
+BROKER_TOKEN=secret-broker-token
```

Comment:
Maybe worth mentioning we have samples for these. Example: https://github.com/snyk/broker/blob/master/client-templates/github-com/.env.sample